app/views/ExposureHistory/History.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+  fireEvent,
+  wait,
+  cleanup,
+  render,
+} from '@testing-library/react-native';
+
+import { calendarDays, toExposureHistory } from '../../exposureHistory';
+import { DateTimeUtils } from '../../helpers';
+import { factories } from '../../factories';
+
+import History from './History';
+
+afterEach(cleanup);
+
+describe('History', () => {
+  it('renders', () => {
+    const exposureHistory = buildBlankExposureHistory();
+
+    const { getByTestId } = render(
+      <History exposureHistory={exposureHistory} />,
+    );
+
+    expect(getByTestId('exposure-history-calendar')).not.toBeNull();
+  });
+
+  describe('when given an exposure history that has a possible exposure', () => {
+    describe('and the user taps the date of that exposure', () => {
+      it("shows a 'Next Steps' button", async () => {
+        const twoDaysAgo = DateTimeUtils.beginningOfDay(
+          DateTimeUtils.daysAgo(2),
+        );
+        const datum = factories.exposureDatum.build({
+          kind: 'Possible',
+          date: twoDaysAgo,
+        });
+        const exposureInfo = {
+          [datum.date]: datum,
+        };
+        const calendar = calendarDays(Date.now(), 21);
+        const exposureHistory = toExposureHistory(exposureInfo, calendar);
+
+        const { queryByTestId, getByTestId } = render(
+          <History exposureHistory={exposureHistory} />,
+        );
+
+        const twoDaysAgoIndicator = getByTestId(`calendar-day-${twoDaysAgo}`);
+
+        expect(queryByTestId('exposure-history-next-steps-button')).toBeNull();
+
+        fireEvent.press(twoDaysAgoIndicator);
+
+        await wait(() => {
+          expect(
+            getByTestId('exposure-history-next-steps-button'),
+          ).not.toBeNull();
+        });
+      });
+    });
+  });
+});
+
+const buildBlankExposureHistory = () => {
+  const datum = factories.exposureDatum.build();
+  const exposureInfo = {
+    [datum.date]: datum,
+  };
+  const calendar = calendarDays(Date.now(), 21);
```

Comment:
Perhaps put `21` in a constant because it's used in multiple places.

Reply:
totally agree. also not a huge fan of how the 21 is floating around in general. I feel its usual to keep separate, but it feels like there wants to be a function that pre-applies this.